nemo_skills/pipeline/eval.py

```diff
@@ -190,34 +226,43 @@ def eval(
 
     eval_cmds = (
         [
-            get_greedy_cmd(
+            cmd
+            for benchmark in benchmarks.keys()
+            for cmd in get_greedy_cmd(
                 benchmark,
                 split,
                 output_dir,
                 extra_eval_args=extra_eval_args,
                 extra_arguments=extra_arguments,
                 extra_datasets=extra_datasets,
+                num_chunks=num_chunks,
+                chunk_ids=chunk_ids,
             )
-            for benchmark in benchmarks.keys()
         ]
         if not skip_greedy
         else []
     )
     eval_cmds += [
-        get_sampling_cmd(
+        cmd
+        for benchmark, rs_num in benchmarks.items()
+        for rs in range(starting_seed, starting_seed + rs_num)
+        for cmd in get_sampling_cmd(
```

Comment:
Its a bit confusing what exactly this list is going to look like. Maybe better to use normal triple loop for clarity.

Reply:
Sounds good, let me do it in a follow-up PR (the part probably needs a larger refactoring)